Remotion/Web/Development.WebTesting/BrowserSession/BrowserSessionBase.cs

```diff
@@ -87,9 +92,32 @@ protected T BrowserConfiguration
       get { return _browserConfiguration; }
     }
 
-    public void AcceptModalDialog (Options? options = null)
+    public TimeSpan DefaultBidiTimeout => TimeSpan.FromSeconds(1);
```

Comment:
```suggestion
    public TimeSpan DefaultBidiTimeout => _biditimeout;
```

Unless I'm missing something?